riscv-asm.md

```diff
@@ -793,7 +793,7 @@ the current position:
 
 [^3]: `ra` is implicitly used to save the return address.
 [^4]: similar to `call <symbol>`, but `<rd>` is used to save the return address instead.
-[^5]: `t1` is implicitly used as a scratch register.
+[^5]: `t1` is implicitly used as a scratch register, but if the `Zicfilp` extension is enabled, it will use `t2` instead.
```

Comment:
I don't think I like this sentence, but I'm struggling to make it better.

Maybe

`If the `Zicfilp` extension is enabled, `t2` is implicitly used as a scratch register. Otherwise, `t1` is implicitly used as a scratch register.`